mulog-kinesis/test/com/brunobonacci/mulog/publishers/test_publisher.clj

```diff
@@ -0,0 +1,56 @@
+(ns com.brunobonacci.mulog.publishers.test-publisher
+  (:require [com.brunobonacci.mulog :as μ]
+            [cheshire.core :as json]
+            [cognitect.aws.client.api :as aws])
+  (:import (java.util.concurrent TimeUnit)))
+
+(def KINESIS-LOCAL-STREAM-NAME "mulog-test-stream")
+
+(def KINESIS-LOCAL-SETTINGS {:api :kinesis
+                             :endpoint-override
+                                  {:protocol :http
+                                   :hostname "localhost"
+                                   :port 4568}})
+
+(defmacro with-local-kinesis-client
+  []
+  `(aws/client KINESIS-LOCAL-SETTINGS))
+
+(defmacro with-local-kinesis-stream
+  [client op params]
+  `(let [rq# (merge ~params {:StreamName KINESIS-LOCAL-STREAM-NAME})]
+     (aws/invoke ~client {:op ~op :request rq#})))
```

Comment:
same here, no need for `with-`  regarding the name this seems to be a wrapper for a kinesis operation so maybe `kinesis-invoke` could be an appropriate name